executors/executor.go

```diff
@@ -26,6 +26,7 @@ type Executor interface {
 	VolumeExpand(host string, volume *VolumeRequest) (*Volume, error)
 	VolumeReplaceBrick(host string, volume string, oldBrick *BrickInfo, newBrick *BrickInfo) error
 	VolumeInfo(host string, volume string) (*Volume, error)
+	HealInfo(host string, volume string) (*HealInfo, error)
```

Comment:
@jarrpa I checked it again, and since this is information about the system, it could be that at some time we look at other values in the structure. 

I remove this request. It is ok as it is.